src/main/java/core/basesyntax/dao/WriteBalanceImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.dao;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class WriteBalanceImpl implements WriteBalance {
+    @Override
+    public boolean write(String balance) throws IOException {
+            File csvFile = new File("src/main/resources/balance.csv");
+            FileWriter fileWriter = new FileWriter(csvFile);
+            fileWriter.write(balance);
+            fileWriter.close();
+        return true;
+    }
```

Comment:
Checked exception should be handled 
Do you need to return boolean? only return type is true

```suggestion
    public boolean write(String balance) {
            File csvFile = new File("src/main/resources/balance.csv");
            FileWriter fileWriter = new FileWriter(csvFile);
            fileWriter.write(balance);
            fileWriter.close();
    }
```